hack/getting_started.sh

```diff
@@ -0,0 +1,563 @@
+#!/bin/bash
+function isCmdInstalled() {
+    local cmd=$1
+    which ${cmd}
```

Comment:
Recommend : command -v over which -> which is external to shell and is divergent when used across platforms, command is built-in
`$ which command
command: shell built-in command
$ command -v command
command
$ which docker
/usr/local/bin/docker
$ command -v docker
/usr/local/bin/docker
`
The above variability is *within* one platform in "which" output and hence not consistent